server/protocols/webadmin/webadmin-data/src/main/java/org/apache/james/webadmin/routes/MappingRoutes.java

```diff
@@ -85,10 +85,8 @@ public void define(Service service) {
                 .entrySet()
                 .stream()
                 .flatMap(entry -> entry.getValue().asStream()
-                    .map(mapping -> Pair.of(entry.getKey(), mapping)))
-                .collect(Guavate.toImmutableListMultimap(
-                    pair -> pair.getLeft().asString(),
-                    pair -> MappingValueDTO.fromMapping(pair.getRight())));
+                    .map(mapping -> Pair.of(entry.getKey().asString(), MappingValueDTO.fromMapping(mapping))))
```

Comment:
you probably don't need that step, you probably can put these transformations into the call to toImmutableListMultimap

Reply:
I still need map to get access to MappingSource. If I put these transformations into toImmutableListMultimap, then I can't. Could you show me how to do it in detail?

Reply:
BTW I tried, I still need to use the `.map()`

Reply:
ok